README.md

```diff
@@ -100,41 +127,47 @@ Each line is a single event collected by Tracee, with the following information:
-4. PPID - parent pid of the calling process
+For a complete list of trace options, run `--trace help`.
 
+### Capturing artifacts
 
-### Configuration flags
+Tracee has a unique feature that lets you capture interesting artifacts from running applications, using the `--capture` flag.
 
-- Use `--help` to see a full description of all options.
-Here are a few commonly useful flags:
-- `--trace` Sets the trace mode. For more information see [Trace Mode Configuration](#Trace-Mode-Configuration) below
-- `--event` allows you to specify a specific event to trace. You can use this flag multiple times, for example `--event execve --event openat`.
-- `--list` lists the events available for tracing, which you can provide to the `--event` flag.
-- `--output` lets you control the output format, for example `--output json` will output as JSON lines instead of table.
-- `--capture` capture artifacts that were written, executed or found suspicious, and save them to the output directory. Possible values are: 'write'/'exec'/'mem'/'all'
+All captured artifacts are saved in Tracee's "output directory" which can be configured using `--capture dir:/path/to/dir`.
 
-### Trace Mode Configuration
+Tracee can capture the following types of IoCs:
 
-`--trace` and `-t` set whether to trace events based upon system-wide processes, or Containers. It also used to set whether to trace only new processes/containers (default), existing processes/containers, or specific processes.
-Tracing specific containers is currently not possible. The possible options are:
+1. Written files: Anytime a file is being written to, the contents of the file will be captured. This can be filtered to select specific file patterns to capture.
+2. Executed files: Anytime a binary is being executed, the binary file will be captured. If the same binary is executed multiple times, it will be captured just once.
+3. Memory files: Anytime a "memory unpacker" is detected, the suspicious memory region will be captured. This is triggered when memory protection changes from Write+Execute to Write.
 
-Option | Flag(s):
---- | --- |
-Trace new processes (default) | no `--trace` flag, `--trace p`, `--trace process` or `--trace process:new`
-Trace existing and new processes | `--trace process:all`
-Trace specific PIDs | `--trace process:<pid>,<pid2>,...` or `--trace p:<pid>,<pid2>,...`
-Trace filtered process and all of its children | `--trace process:follow`
-Trace new containers | `--trace c`, `--trace container` or `--trace container:new`
-Trace existing and new containers | `--trace container:all`
-Trace new processes not in a container | `--trace h`, `--trace host` or `--trace host:new`
-Trace all processes not in a container | `--trace host:all`
+To use, `--capture exec`, `--capture mem`, and `--capture write` capture executed, memory, and written files respectively. 
+Written files can be filtered using an optional path prefix, e.g `--capture write=/etc/*` will capture anything written blow `/etc/`.
```

Comment:
If you are writing about the write filter here, I don't think it is necessary in point 1 above